src/Sql/Platform/Mysql/Ddl/CreateTableDecorator.php

```diff
@@ -163,14 +163,13 @@ private function normalizeColumnOption($name)
     }
 
     /**
-     * @internal @private this method is only public for PHP 5.3 compatibility purposes.
      *
      * @param string $columnA
      * @param string $columnB
      *
      * @return int
      */
-    public function compareColumnOptions($columnA, $columnB)
+    private function compareColumnOptions($columnA, $columnB)
```

Comment:
I would instead make this protected, however, this is a BC break in the event someone extended the class.  You also cannot change visibility.  I would revert this portion of the commit so that it can be merged to master and subsequently make this change against the develop branch as a secondary PR.